crud/select.lua

```diff
@@ -22,29 +23,36 @@ local select_module = {}
 
 local SELECT_FUNC_NAME = '__select'
 
-local function call_select_on_storage(space_name, conditions, opts)
-    checks('string', '?table', {
-        limit = '?number',
+local function call_select_on_storage(space_name, index_id, scan_value, iter, conditions, opts)
```

Comment:
As for me "iter" should be part of opts

`box.space[*space_name*].index[*index_id*]:pairs(*scan_value*, *{<options included iter>}*)`. Or it could be "scan_opts" if you want separate it from batch_size/after_tuple.